deployment/docker/victorialogs/filebeat/README.md

```diff
@@ -16,39 +19,14 @@ docker compose rm -f
 The docker compose file contains the following components:
 
 * filebeat - fileabeat is configured to collect logs from the `docker`, you can find configuration in the `filebeat.yml`. It writes data in VictoriaLogs
-* filebeat-exporter - it export metrics about the filebeat
 * VictoriaLogs - the log database, it accepts the data from `filebeat` by elastic protocol
 * VictoriaMetrics - collects metrics from `filebeat` via `filebeat-exporter`, `VictoriaLogs` and `VictoriaMetrics`
-* grafana - it comes with two predefined dashboards for `VictoriaLogs` and `VictoriaMetrics`
 
 Querying the data 
 
 * [vmui](https://docs.victoriametrics.com/victorialogs/querying/#vmui) - a web UI is accessible by `http://localhost:9428/select/vmui`
 * for querying the data via command-line please check [these docs](https://docs.victoriametrics.com/victorialogs/querying/#command-line)
 
-the example of filebeat configuration(`filebeat.yml`)
-
-```yaml
-filebeat.autodiscover:
-  providers:
-    - type: docker
-      hints.enabled: true
-
-processors:
-  - add_docker_metadata: ~
-
-output.elasticsearch:
-  hosts: [ "http://victorialogs:9428/insert/elasticsearch/" ]
-  worker: 5
-  parameters:
-    _msg_field: "message"
-    _time_field: "@timestamp"
-    _stream_fields: "container.name"
-
-http:
-  enabled: true
-  host: 0.0.0.0
-  port: 5066
-```
+the example of [filebeat configuration](./syslog./filebeat.yml)
```

Comment:
do you hting this is a good way to hide the config? what's the point of this? currently user just can copy it from here otherwise he must open the file

Reply:
github supports permalinks rendering
you can check result here
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/6eedf20d744f5017b0e6eef438eedc8b14084e8d/deployment/docker/victorialogs/fluentbit/README.md